src/Edit.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import './App.css';
+import {Link} from  'react-router-dom';
+
+function EditInterview(event) {
+  
+  const pageUrl = window.location.href
+  const id = pageUrl.substring(35,38);
+  
+  const perform_function = async(json_object) => {
+    console.log(json_object);
+    
+    const options = 
+      {
+          method: 'PATCH',            
+          mode: 'cors',
+          headers: {
+              'Content-Type': 'application/json'
+          },
+          body: JSON.stringify(json_object),
+      };
+      const response = await fetch(`http://localhost:3000/interviews/` + id, options);
+      console.log(response);
+      window.location = "/";
+
+  }
+
+
+  console.log("I am in EditInterview ");  
+  event.preventDefault();
+  const data = new FormData(event.target);
+  var json_object   = {}
+  data.forEach((value, key) => {json_object[key] = value});
+
+  perform_function(json_object);
+  
+  
+}
+
+export default class Edit extends React.Component{

```

Comment:
Not a good name. Please use a different one

Reply:
made EditEvent